Resources/Prototypes/Entities/Clothing/Hands/gloves.yml

```diff
@@ -46,6 +47,30 @@
     fiberColor: fibers-blue
   - type: FingerprintMask
 
+# April Fools: Vox boxing gloves, given to all Vox. A bit dangerous.
+- type: entity
+  parent: ClothingHandsGlovesBoxingRed
+  id: ClothingHandsGlovesBoxingVox
+  name: vox boxing gloves
+  description: These pack a punch! They'll never make you eat lunch in the maints again! SCRAWWWW!!
+  components:
+  - type: Sprite
+    sprite: Clothing/Hands/Gloves/Boxing/boxingblue.rsi
+    state: icon
+  - type: Clothing
+    sprite: Clothing/Hands/Gloves/Boxing/boxingblue.rsi
+  - type: Fiber
+    fiberMaterial: fibers-leather
+    fiberColor: fibers-blue
+  - type: FingerprintMask
+  - type: MeleeWeapon
+    attackRate: 2
+    animation: WeaponArcBox
+    damage:
+      types:
+        Blunt: 6 # Double Vox unarmed attack
```

Comment:
I have no idea if this would be busted but it's a shitday event so I'm not sure it's the end of the world